Sources/Commands/PackageTools/SwiftPackageTool.swift

```diff
@@ -41,6 +41,9 @@ public struct SwiftPackageTool: ParsableCommand {
             Init.self,
             Format.self,
 
+            Install.self,
+            Remove.self,
```

Comment:
follow up to the comment on command naming above
```suggestion
            Uninstall.self,
```